### ISSUE TITLE
CI: show off kaitai-struct-compiler unit tests

### ISSUE THREAD

**Issue by GreyCat** [kaitai-io/kaitai_struct#1057]
KSC includes unit tests, which are sometimes known as `translator-tests`. These technically are [ran during CI](https://github.com/kaitai-io/kaitai_struct/blob/2c37cb90165c737cd52d72bec42585266f408b38/.github/workflows/main.yml#L53-L56), but they known to be not 100% complete, and thus they don't break the build, but it's too hard to keep track of them at the moment.

It would be great to publish these results somewhere around https://ci.kaitai.io/ (maybe on a separate page?) and keep track of the progress.

**Comment by generalmimon** [comment]
Isn't this a duplicate of https://github.com/kaitai-io/kaitai_struct/issues/551?

Perhaps this issue is meant more generally, though - https://github.com/kaitai-io/kaitai_struct/issues/551 has specifically the expression tests in mind, which are part of the internal compiler test suite, but are not the only tests there.

**Comment by GreyCat** [comment]
Yes and no. #551 speaks of expression translator tests, which clearly form a test-language matrix. Some language's implementations might be broken sometimes, and that's what we want to track.

This one is more generic, showcasing the rest of the tests (probably excluding expression translator tests that will be on a separate page with language matrix layout), just something like more regular "test → status" layout.
